roles/third_party/ldap-install/templates/users.ldif.j2

```diff
@@ -13,7 +13,7 @@ givenName: Joe
 mail: {{ __ldap_userid }}1@{{ __ldap_user_mail_domain }}
 uid: {{ __ldap_userid }}1
 description: I am interested in Robotics; I play Hockey and Soccer
-userPassword: {{ __ldap_user_admin_password }}
+userPassword: {{ __ldap_user_password }}
```

Comment:
Sorry, I missed this one - those are also two different passwords - in theory they can be the same, but in general the shouldn't?